app/main/controllers/template/rtmedia-functions.php

```diff
@@ -578,18 +578,26 @@ function rtmedia_media( $size_flag = true, $echo = true, $media_size = 'rt_media
 	if ( isset( $rtmedia_media->media_type ) ) {
 
 		if ( 'photo' === $rtmedia_media->media_type ) {
-
 			$src = wp_get_attachment_image_src( $rtmedia_media->media_id, $media_size );
 
-			/**
-			 * Used `set_url_scheme` because `esc_url` breaks the image if there is special characters are there into image name.
-			 * Added by checking the code from "wp-admin/includes/media.php:2740".
-			 * Because in media library, it was not breaking.
-			 *
-			 * Add timestamp to resolve conflict with cache image.
-			 */
-			$html = "<img src='" . set_url_scheme( $src[0] . '?' . time() ) . "' alt='" . esc_attr( $rtmedia_media->post_name ) . "' />";
+			if ( ! empty( $src[0] ) ) {
```

Comment:
@SID177 Please check ! empty( $src) else it will notice that array index

Reply:
@umeshnevase It won't throw any notice, `empty` will handle checking `$src` as well as `$src[0]`.
You can try
```
$a = null;
echo !empty($a[0])?'yes':'no';
```
This code in online PHP interpreter and it'll output `no`.